### PR TITLE
birth_year should be birth_second

### PR DESCRIPTION
**Pull request by Mason11987**

### CODE REVIEW

**Comment by myk002** [comment]
Looks straightforward. Could you add a changelog entry for the fix?

**Comment by Mason11987** [comment]
Been a while since I contributed here.  What does that look like?  Thanks

**Comment by myk002** [comment]
It's the https://github.com/DFHack/dfhack/blob/e1e3c6f7ace2010b181d63dd75d9179b504221b7/docs/changelog.txt#L36 file

Add a line to the "fixes" section

**Comment by Mason11987** [comment]
Done, thanks!

File: changelog.txt [hunk 1]
```diff
@@ -25,6 +25,7 @@ that repo.
 - `quickfort`: fix incorrect directions for tracks that extend south or east from a track segment pair specified with expansion syntax (e.g. T(4x4))
 - `quickfort`: fix parsing of multi-part extended zone configs (e.g. when you set custom supply limits for hospital zones AND set custom flags for a pond)
 - `quickfort`: fix error when attempting to set a custom limit for plaster powder in a hospital zone
+- `exportlegneds`: fix issue where birth year was outputted as birth seconds
```
Comment (lethosor) [inline changelog.txt:28]: Build failed due to a typo: https://github.com/DFHack/scripts/runs/4071829687?check_suite_focus=true

> ```
> ../../../dfhack/docs/_auto/news.rst:9: WARNING: undefined label: exportlegneds
> ```

```suggestion
- `exportlegends`: fix issue where birth year was outputted as birth seconds
```

**Pull request merged**